content/post/setup-kubernetes-raspberry-pi-cluster.md

```diff
@@ -1,45 +1,45 @@
-
-So, Kubernetes seems to be a compelling competitor to Docker. Let's get our hands on it!
+  - **Lightweight:** Software should be fit on a Raspberry Pi or smaller. As proofed in my talk above, Kubernetes painlessly runs on a Raspberry Pi.
+  - **ARM compatible:** Since the ARM CPU architecture is designed for low energy consumption but still able to deliver a decent portion of power, the Raspberry Pi runs an ARM CPU. Thanks to Lucas, Kubernetes is ARM compatible.
+  - **General purpose:** Hadoop or Apache Spark are great for data analysis. But what if your use case changes? We prefer general purpose software that allows to run **anything**. Kubernetes uses a container runtime (with Docker as the 100% supported runtime for the time being) that allows to run whatever you want.
+  - **Production ready:** Since we compare Kubernetes against a production ready Docker suite, let's be fair and only choose equivalents. Kubernetes itself is production ready, and while the ARM port has some small issues, it's working exactly as expected when going the official kubeadm route, which also will mature with time.
 
+So, Kubernetes seems to be a compelling competitor to Docker Swarm. Let's get our hands on it!
 
 Wait – what about Kubernetes-on-ARM?
 ---------------
-If you followed the discussion of K8s on ARM for some time, you probably know about Lucas' project [kubernetes-on-arm](https://github.com/luxas/kubernetes-on-arm). Since the beginning of the movement to bring K8s on ARM in 2015, this project has always been the most stable and updated.
+If you followed the discussion of Kubernetes on ARM for some time, you probably know about Lucas' project [kubernetes-on-arm](https://github.com/luxas/kubernetes-on-arm). Since the beginning of the movement to bring Kubernetes on ARM in 2015, this project has always been the most stable and updated.
 
-However, now Lucas' contributions have successfully been merged into official K8s repositories, such that there is point any more for using the kubernetes-on-arm project. In fact, the features of that project are far behind of what's now implemented in the official repos.
+However, during 2016, Lucas' contributions have successfully been merged into official Kubernetes repositories, such that there is no point any more for using the kubernetes-on-arm project. 
+In fact, the features of that project are far behind of what's now implemented in the official repos, **and that has been Lucas' goal from the beginning.**
 
-So if you're up to using K8s, please stick to the official repos now. And as of documentation, the following setup how-to can be considered as official for K8s on ARM.
+So if you're up to using Kubernetes, please stick to the official repos now. And as of the kubeadm documentation, the following setup is be considered official for Kubernetes on ARM.
```

Comment:
I think that "is be" is grammatically incorrect. This should be "is considered".